module/category/src/Domain/ValueObject/CategoryCode.php

```diff
@@ -53,6 +53,7 @@ public function __toString(): string
      */
     public static function isValid(string $value): bool
     {
-        return strlen($value) <= 255;
+        return '' !== $value
+            && strlen($value) < 256;
```

Comment:
what if `$value = '  ' `?

Reply:
You should use trim and then check it. Empty function is too risky.